lib/engine/config/headway.ex

```diff
@@ -32,9 +32,11 @@ defmodule Engine.Config.Headway do
   def from_map(_, _, _), do: :error
 
   @spec current_time_period(DateTime.t()) :: time_period()
-  def current_time_period(dt) do
-    # Subtract 3 hours, since the service day ends at 3 AM
-    day_of_week = DateTime.add(dt, -3, :hour) |> Date.day_of_week()
+  def current_time_period(dt_to_shift) do
+    # Subtract 4 hours, since the service day ends at 4 AM
+    dt = DateTime.shift_zone!(dt_to_shift, "America/New_York")
```

Comment:
This should use the existing `:time_zone` app config key instead of duplicating the `"America/New_York"`.